src/libraries/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -20,6 +20,7 @@
     <Compile Include="JsonElementWriteTests.cs" />
     <Compile Include="JsonEncodedTextTests.cs" />
     <Compile Include="JsonGuidTestData.cs" />
+    <Compile Include="JsonNodeOptionsTests.cs" />
```

Comment:
nit: Move this to the other ItemGroup which contains all the `JsonNode` related test files. We are keeping them separate until the API shape has been reviewed/finalized to make it easier to track/update.

https://github.com/dotnet/runtime/blob/f3be2149c44400a3fefbc7055240a6a9169c58cd/src/libraries/System.Text.Json/tests/System.Text.Json.Tests.csproj#L133

Reply:
Done. I will fix the other suggestion soon.